README.md

```diff
@@ -1,54 +1,91 @@
 # Rails Static Request
 
-Objectives
+## Routing
 
-1. Define the purposes of routing, the mapping between an HTTP request and a specific controller action in your application.
-2. Identify route errors from Rails (going to a url with no route) and how to solve it (by drawing a route)
-3. Draw a route with get, mapping a url to a controller action
-4. Create a new controller by hand and inherit from application controller
-5. Define a controller action by defining a method in a controller
-6. Identify missing template errors from Rails (going to an action with no template) and how to solve it (by creating a template)
-7. Use render to explicity render a template
-8. Define the Reails implicit rendering convention
-9. Define the steps required to create a static request
+How does your application know what view to render to users? This is where routing comes in, as a framework Rails has a comprehensive routing system for both dynamic and static pages. Below are the differences between a static and dynamic route:
 
+* **Static route** - A static route will render a view that does not change, you typically will not send parameters to it, examples would be: a site's about or contact pages.
+* **Dynamic route** - Dynamic routes are pages that accept parameters and render different content based on those parameters, an example would be: a blog's post page that contains a specific article.
 
-## Notes
+In this lesson we are going to specifically cover static pages to ensure that you can get a firm understanding of how routing works in a Rails application.
 
-Should explain that every URL that we want our application to handle must map to a specific controller and action.
+Before we dive into the code and routing configurations, it helps to know how HTTP works at a high level. Below is the flow that takes place when a user attempts to go to a page on a Rails application:
 
-When a request comes into our application, our router must pick a controller and a method in that controller that is responsible for generating a response. We've taught a lot of this in Sinatra so it can be a bit light here.
+1. A URL is entered into the browser, this is the HTTP request
+2. That request is sent to the server where the application's router interprets the request and sends a message to the controller mapped to that route
+3. The controller communicates with the view file mapped to the controller method
+4. The server returns that HTTP response, which contains the view and page that can be viewed in the browser
 
-Use the example of how we want to have an 'about' page on our site. So let's try to go there, let's just type localhost:3000/about.
+## Implementing a Static Route
 
-We get a routing error. Let's draw the route.
+Let's try this out in our application, I'm going to continue using the ```blog-flash``` application that I walked through in the Rails Basic lesson.
```

Comment:
As much as possible, we try and reduce mentioning previous lessons. This is mostly to reduce dependencies. Similar to code, dependecies are fine...as long as they are worth the effort required to maintain them. It's a bit annoying but unless it's suuuuuper annoying I'd go with something like "Let's work this out in an example blog app. You can find the starter code in this repo"